### DISCUSSION TITLE
feat: Navigable Tabs

### DISCUSSION THREAD

**Discussion by mattgoedde** [microsoft/fluentui-blazor#452] [Ideas]
# 🙋 Feature Request
Assign each `<FluentTabs>` tab to a Blazor page. This would allow users to bookmark a tab and navigate forwards and backwards with their browser's buttons. The `<FluentTabs>` component would be similar to a nested layout: see [ASP.NET Core Blazor layouts](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/layouts?view=aspnetcore-7.0#nested-layouts)

# 🤔 Expected Behavior
- each `<FluentTab>` have an `href` attribute
- clicking a tab should navigate to a different Blazor page
- each different Blazor page should have content in the `<FluentTabPanel>`
- back/forward buttons in browser should work on tabs
- visiting the URL of the tab should activate correct tab

**Comment by vnbaaij** [comment]
We will not have resources to do this anytime soon. We do however take PR's 😉
